pulp_ansible/tests/functional/constants.py

```diff
@@ -38,11 +40,13 @@
 # FIXME: replace this with the location of one specific content unit of your choosing
 ANSIBLE_URL = urljoin(ANSIBLE_FIXTURE_URL, "")
 
-ANSIBLE_GALAXY_COLLECTION_URL = "https://galaxy-dev.ansible.com"
+ANSIBLE_GALAXY_COLLECTION_URL = "https://galaxy.ansible.com/api/v2/collections/"
+
+ANSIBLE_COLLECTION_FIXTURE_URL = urljoin(ANSIBLE_GALAXY_COLLECTION_URL, NAMESPACE_TESTING)
 
 ANSIBLE_COLLECTION_REMOTE_PATH = urljoin(BASE_REMOTE_PATH, "ansible/collection/")
 
-COLLECTION_WHITELIST = "testing.ansible_testing_content"
+COLLECTION_WHITELIST = "testing.k8s_demo_collection"
```

Comment:
@fabricio-aguiar, any reason to update this one?

Reply:
`testing.ansible_testing_content` isn't available in production galaxy. we switched to production galaxy because it has less roles and we don't have any options for filtering ATM.

Reply:
@daviddavis, thanks.